pkgs/applications/audio/ir.lv2/default.nix

```
@@ -1,25 +1,24 @@
-{ lib, stdenv, fetchFromGitHub, fftw, gtk2, lv2, libsamplerate, libsndfile, pkg-config, zita-convolver }:
+{ lib, stdenv, fetchgit, fftw, gtk2, lv2, libsamplerate, libsndfile, pkg-config, zita-convolver }:
 
 stdenv.mkDerivation rec {
   pname = "ir.lv2";
-  version = "1.2.4";
+  version = "unstable-2018-06-21";
```

```suggestion
  version = "0-unstable-2018-06-21";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming

---

Thanks for the heads up!